### ISSUE TITLE
Deploy button fails with `apps_tasklog_pkey` constraint violation

### ISSUE THREAD

**Issue by alecgorge** [palfrey/wharf#65]
Rebuilding works just fine, but a deploy does not work. Could this be cause by a private Github repo used as `GITHUB_URL`? What is the proper way to add authentication?

```
Task id: 7beba0fa-4c11-4847-b06b-5ab2e39f1d01
Task state: FAILURE

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
psycopg2.IntegrityError: duplicate key value violates unique constraint "apps_tasklog_pkey"
DETAIL:  Key (task_id)=(7beba0fa-4c11-4847-b06b-5ab2e39f1d01) already exists.


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 375, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 632, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/wharf/tasks.py", line 120, in deploy
    description="Deploying %s" % app_name
  File "/usr/local/lib/python3.6/site-packages/django/db/models/base.py", line 718, in save
    force_update=force_update, update_fields=update_fields)
  File "/usr/local/lib/python3.6/site-packages/django/db/models/base.py", line 748, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/usr/local/lib/python3.6/site-packages/django/db/models/base.py", line 831, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/usr/local/lib/python3.6/site-packages/django/db/models/base.py", line 869, in _do_insert
    using=using, raw=raw)
  File "/usr/local/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/django/db/models/query.py", line 1136, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/usr/local/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1289, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: duplicate key value violates unique constraint "apps_tasklog_pkey"
DETAIL:  Key (task_id)=(7beba0fa-4c11-4847-b06b-5ab2e39f1d01) already exists.
```

**Comment by palfrey** [comment]
Something of a weird one. So, I wasn't able to reproduce on my local machine, but was eventually able to make work on my deployed box. Once I went into the code there was a certain level of "how did this ever work?", so I think I was able to fix this in #66. Haven't got an automated test for it yet, but you should be able to do things again now.

**Comment by palfrey** [comment]
Please reopen this if you're still seeing the issue
